src/plugins/home/public/services/feature_catalogue/feature_catalogue_registry.ts

```diff
@@ -56,8 +57,9 @@ export interface FeatureCatalogueSolution {
 
 export class FeatureCatalogueRegistry {
   private capabilities: Capabilities | null = null;
-  private readonly features = new Map<string, FeatureCatalogueEntry>();
-  private readonly solutions = new Map<string, FeatureCatalogueSolution>();
+  private solutions = new Map<string, FeatureCatalogueSolution>();
+  private features = new Map<string, FeatureCatalogueEntry>();
```

Comment:
nit: instead of updating the state in two places, I think it would be safer to keep it only inside the `BehaviorSubject`

Reply:
Agreed. 👍  Probably keep it in the Subject and then call `subject.value` where needed.

Reply:
Sure thing! Thanks for the hint about `subject.value` 👍🏻